docs/contracts/digital-asset.md

```diff
@@ -0,0 +1,315 @@
+---
+title: LSP7-Digital Asset
+sidebar_position: 5
+---
+
+# LSP7-Digital Asset
+
+The **DigitalAsset** contract represents digital assets, for either fungible or non-fungible tokens where minting and transfering is specified with an amount of tokens. It have some functions from **[ERC20](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol)** and **[ERC777](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC777/ERC777.sol)** with more upgraded features.
+
+This contract serve as a **Fungible Token Contract** when  `isNFT`  bool is marked as **FALSE** in the constructor, and serve as **Non-Fungible Token Contract** otherwise. 
+
+:::note
+**_DigitalAsset implementation contains the methods from [ERC165](https://eips.ethereum.org/EIPS/eip-165) and [ERC725Y](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-725.md#erc725y) contract to attach metadata to the asset._**
+:::
+
```

Comment:
Do we really need this with a big callout UI? Takes lot of attention on the page.